Automattic/jetpack
# Social | Initial State: Migrate URLs on Social admin page

Extracted out of #38983 to split it into smaller PRs.

## Proposed changes:

* Migrate URLs on the Social admin page to the new script data

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Activate Social plugin
* Goto Social admin page
* Confirm that the license activation link works correctly
    <img width="482" alt="Screenshot 2024-10-17 at 10 54 03 AM" src="https://github.com/user-attachments/assets/f699f119-0d57-43d1-9e30-12ab57c9b191">
* Confirm that "Create a note" link works correctly
    <img width="423" alt="image" src="https://github.com/user-attachments/assets/decb89ed-0ba6-43bb-8c5f-e5fa88bf7464">
* There are also the changes to connection management URLs which point to Calypso but those URLs are no longer used but if you want to test them, you can hardcode to negate `useAdminUiV1` flag in the front-end code.
* Verify that editor loads fine on Simple and Atomic sites
